pkg/cluster/spec/spec_manager.go

```diff
@@ -99,7 +100,13 @@ func (s *SpecManager) SaveMeta(clusterName string, meta Metadata) error {
 	}
 
 	data, err := yaml.Marshal(meta)
-	if err != nil {
+
+	// remove quotes
```

Comment:
`Marshal()` converts the object to **valid** YAML bytes, so it's not a good idea to replace anything after it.

If the processing methods are all proper when building the `meta` object, it should be no invalid quote chars in the object.